main.py

```diff
@@ -1,7 +1,44 @@
+from collections import namedtuple  # noqa: F401
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    # Genre.objects.create(name="Western")
+    # Genre.objects.create(name="Action")
+    # Genre.objects.create(name="Dramma")
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
```

Comment:
The name 'Kianu Reaves' seems to be a typo. It should be 'Keanu Reeves'. This is corrected later in the script, but it's better to use the correct name initially to avoid unnecessary updates.